packages/experiments/src/components/Button/Button.styles.ts

```diff
@@ -37,8 +43,8 @@ const circularTokens: IButtonComponent['tokens'] = (props, theme): IButtonTokenR
 const enabledTokens: IButtonComponent['tokens'] = (props, theme): IButtonTokenReturnType => {
   const { semanticColors } = theme;
   return {
-    backgroundColor: semanticColors.buttonBackground,
-    backgroundColorHovered: semanticColors.buttonBackgroundHovered,
+    backgroundColor: semanticColors.inputBackground,
+    backgroundColorHovered: semanticColors.buttonBackground,
```

Comment:
We need to make a pass for all semantic colors and fix the mapping to the palette colors.
I think for now it's better to use the palette colors vs semanticColors because semantically it's not entirely correct what you did here. You just picked a semantic slot to match the palette color you need.

Reply:
Done